### ISSUE TITLE
Learning rate issues

### ISSUE THREAD

**Issue by boborbt** [ml-unito/fair-networks#7]
Current experiments show that the loss for the 's' part of the network behaves in a very different way w.r.t. the y-loss. 

It is to be investigate if:
- if it is due to the way 's' loss is evaluated for tensor board (it is a bit tricky since 's' is trained separately from the rest of the network).
- if using two different learning weights helps.
